src/app/dashboard/store-form.tsx

```diff
@@ -14,46 +14,49 @@ import {
 } from "@/components/ui/form";
 import { Button } from "@/components/ui/button";
 import { MerchantInfo } from "../dto/merchant";
-import axios from "axios";
-import { saveInfo } from "../services/merchant/merchant";
+import { UploadButton } from "@/utils/uploadthing";
+import { useState } from "react";
 
 const formSchema = z.object({
-  storeName: z.string().min(2).max(50),
-  storeUrl: z.string().url().optional(),
-  storeDescription: z.string().min(2).max(100),
-  storeLogo: z.string().min(0),
+  name: z.string().min(2).max(50),
+  url: z.string().url().optional(),
+  description: z.string().min(2).max(100),
+  profileImage: z.string().min(0).optional(),
 });
 
 export function StoreForm() {
+  const [imageUrl, setImageUrl] = useState<string | undefined>(undefined);
```

Comment:
Non si può usare lo state del form?

Reply:
Mh, si. Cerco di farlo

Reply:
EDIT: avevo letto male. In che senso lo state del form?

Reply:
Ah ok. Ho fatto 😅